<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="default">
    <gui>
      <camera name="user_camera">
        <pose>-5 0 1 0 0.2 0</pose>
      </camera>
    </gui>
    <physics type="ode">
      <ode>
        <solver>
          <type>quick</type>
          <iters>100</iters>
          <sor>1.0</sor>
        </solver>
        <constraints>
          <cfm>0.0</cfm>
          <erp>0.2</erp>
          <contact_max_correcting_vel>0.1</contact_max_correcting_vel>
          <contact_surface_layer>0.0</contact_surface_layer>
        </constraints>
      </ode>
      <real_time_update_rate>-1</real_time_update_rate>
      <!--<max_step_size>0.0020</max_step_size>-->
    </physics>
    <gravity>0 0 -9.8</gravity>

    <include>
      <uri>model://sun</uri>
    </include>

    <model name="sf03">
      <static>true</static>
      <pose>0 0 0 1.57 0 0</pose>

      <link name="sf03">
        <visual name="visual">
          <geometry>
            <mesh>
              <uri>file:///home/lmj/catkin_ws_phd/src/pathplanninginradiationfield/meshes/sf03/version_2/sf03.dae</uri>
              <!-- <scale>0.2 0.2 0.2</scale> -->
            </mesh>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
          </material>
        </visual>

        <collision name="collision">
          <geometry>
            <mesh>
              <uri>file:///home/lmj/catkin_ws_phd/src/pathplanninginradiationfield/meshes/sf03/version_2/sf03.dae</uri>
              <!-- <scale>0.2 0.2 0.2</scale> -->
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>100</mu>
                <mu2>50</mu2>
              </ode>
            </friction>
          </surface>
        </collision>
      </link>
    </model>

  </world>
</sdf>
